src/style.css

```diff
@@ -1 +1,127 @@
-/* add styles here */
+@font-face {
+  font-family: "Avenir" sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir" sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
+
+.form{
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.form,
+.search--bar {
+  width: 100%;
+}
+
+.label {
+  display: block;
+  width: 100%;
+  position: relative;
+}
+
+.label::after {
+ background-image: url(images/Search.svg);
+ background-repeat: no-repeat;
+ background-size: contain;
+ content: "";
+ display: block;
+ position: absolute;
+
+}
+
+.form--big,
+.big {
```

Comment:
Do not increase the specification of the selector, find another way, same for all the similar cases in your CSS file